adyenv6core/src/com/adyen/v6/factory/AdyenRequestFactory.java

```diff
@@ -247,6 +248,22 @@ else if (PAYMENT_METHOD_SEPA_DIRECTDEBIT.equals(cartData.getAdyenPaymentMethod()
         return paymentsRequest;
     }
 
+    public PaymentsRequest createPaymentsRequest(final String merchantAccount,
+                                                 final CartData cartData,
+                                                 final PaymentMethodDetails paymentMethodDetails,
+                                                 final RequestInfo requestInfo,
+                                                 final CustomerModel customerModel) {
+        PaymentsRequest paymentsRequest = new PaymentsRequest();
+        updatePaymentRequest(merchantAccount, cartData, requestInfo, customerModel, paymentsRequest);
+
+        paymentsRequest.setPaymentMethod(paymentMethodDetails);
+
+        ApplicationInfo applicationInfo = updateApplicationInfoEcom(paymentsRequest.getApplicationInfo());
+        paymentsRequest.setApplicationInfo(applicationInfo);
```

Comment:
```suggestion
        updateApplicationInfoEcom(paymentsRequest.getApplicationInfo());
```
should this be enough?
it is used here in this way: https://github.com/Adyen/adyen-hybris/blob/34c14bf8cd617c99473283933c197db429765000/adyenv6core/src/com/adyen/v6/factory/AdyenRequestFactory.java#L478

Reply:
edit: I think the line I referenced is wrong, please ignore the suggestion

Reply:
I will check for capture and cancel and will create a separate commit for the fix if needed.